bindings/python/edlib.pyx

```diff
@@ -152,3 +154,98 @@ def align(query, target, mode="NW", task="distance", k=-1, additionalEqualities=
     cedlib.edlibFreeAlignResult(cresult)
 
     return result
+
+
+
+## function to output NICE alignments
+## only 'task=="path"' works
+
+def cigarToNiceAlignment(query, reference, mode="NW", k=-1, additionalEqualities=None):
+    """ Align query with reference using edit distance, and output alignments in NICE format
+    @param {str or bytes or iterable of hashable objects} query, combined with reference must have no more
+           than 256 unique values
+    @param {str or bytes or iterable of hashable objects} reference, combined with query must have no more
+           than 256 unique values
+    @param {string} mode  Optional. Alignment method do be used. Possible values are:
+            - 'NW' for global (default)
+            - 'HW' for infix
+            - 'SHW' for prefix.
+    @param {int} k  Optional. Max edit distance to search for - the lower this value,
+            the faster is calculation. Set to -1 (default) to have no limit on edit distance.
+    @param {list} additionalEqualities  Optional.
+            List of pairs of characters or hashable objects, where each pair defines two values as equal.
+            This way you can extend edlib's definition of equality (which is that each character is equal only
+            to itself).
+            This can be useful e.g. when you want edlib to be case insensitive, or if you want certain
+            characters to act as a wildcards.
+            Set to None (default) if you do not want to extend edlib's default equality definition.
+    @return Dictionary with following fields:
+            {string} cigar  Cigar is a standard format for alignment path.
+                Here we are using extended cigar format, which uses following symbols:
+                Match: '=', Insertion to reference: 'I', Deletion from reference: 'D', Mismatch: 'X'.
+                e.g. cigar of "5=1X1=1I" means "5 matches, 1 mismatch, 1 match, 1 insertion (to reference)".
+            {string} NICE  NICE is the indented pairwise alignment. 
+            {string} query_aln  query_aln is the aligned query sequence from the input 'query'
+            {string} match_aln  match_aln is the pairwise alignment from the parsed cigar
+                The symbol '|' represents a read match '='
+                The symbol '.' represents a read mismtach 'X'
+                The space ' ' represents an insertion 'I' or deletion 'D'
+            {string} ref_aln  ref_aln is the aligned reference sequence from the input 'reference'                
+    """
+
+    ## 'path' - find edit distance, start and end locations and alignment path. (Default)
+
+    result = align(query=query, target=reference, mode=mode, task="path", k=k, additionalEqualities=additionalEqualities)
+
+    ## access locations, a list of a set [(start, end)], from yech1990
+    ## ref_pos = locations[0][0]
+    ref_pos = result["locations"][0][0]
```

Comment:
what is ref_pos -> let's use full words and let's try to be more descriptive.

Reply:
Aha, ref is for reference. Let's stick to "target"!

Reply:
And if we are considering how to call those three string, I would suggest:
target_aligned, alignment_operations, query_aligned. Or something in that direction.

Reply:
> And if we are considering how to call those three string, I would suggest:
target_aligned, alignment_operations, query_aligned. Or something in that direction.

This seems reasonable. I was just using @yech1990 's  variable names. 

I'll use `target` terminology  instead of reference, that's fine.

Reply:
Ok great!